src/Contracts/PharmacyInterface.php

```diff
@@ -12,4 +12,5 @@ public function getAddress();
     public function setAddress($address);
     public function getregisteredStatus();
     public function setregisteredStatus($registeredStatus);
+    public function getCode();
```

Comment:
This is not needed because the interface method is already defined in ValueSetInterface